Packs/CommonScripts/Scripts/AddKeyToList/AddKeyToList_test.py

```diff
@@ -0,0 +1,253 @@
+from AddKeyToList import add_key_to_list_command
```

Comment:
unit tests should contain given-when-then comment to describe the unit test scenario.
\-Given
    \<some input>
\- When
    \<some state>
Relevant also for all the unit tests files.
\- Then
    \<ensure some behviour>